### PR TITLE
Disabling meshes when stopped

### PR DESCRIPTION
**Pull request by PixelsCommander**

### CODE REVIEW

File: demo/index.ts [hunk 1]
```diff
@@ -13,7 +13,26 @@ import {
 
 import { Godrays, defaultConfig } from "../dist/index";
 
-console.log(Godrays);
+var raf = window.requestAnimationFrame;
+var skipProcent = 0.9;
```
Comment (geobalas) [inline demo/index.ts]: `const`

File: demo/index.ts [hunk 2]
```diff
@@ -13,7 +13,26 @@ import {
...
+var rafs = 0;
```
Comment (geobalas) [inline demo/index.ts]: `let`

File: demo/index.ts [hunk 3]
```diff
@@ -13,7 +13,26 @@ import {
 
 import { Godrays, defaultConfig } from "../dist/index";
 
-console.log(Godrays);
+var raf = window.requestAnimationFrame;
```
Comment (geobalas) [inline demo/index.ts]: `const`

File: demo/index.ts [hunk 4]
```diff
@@ -13,7 +13,26 @@ import {
...
+var skipProcent = 0.9;
+var rafs = 0;
+
+window.requestAnimationFrame = (callback: FrameRequestCallback): number => {
```
Comment (geobalas) [inline demo/index.ts]: I know it's only for the demo, but it's still better not to change the method on `window`. Why didn't you create a function that calls `window.requestAnimationFrame`? If someone wants to play with the code in your demo, they may try to use `window.requestAnimationFrame` with unexpected results.

File: demo/index.ts [hunk 5]
```diff
@@ -13,7 +13,26 @@ import {
...
+    var skip = Math.random() < skipProcent;
```
Comment (geobalas) [inline demo/index.ts]: const

Comment (geobalas) [reply demo/index.ts]: It can be even added in `if` without a variable

**Review by geobalas — CHANGES_REQUESTED** [review]

**Comment by PixelsCommander** [comment]
@geobalas Thanks for reviewing this! It was indeed some code hacking RAF which should not be there. Removed.

**Review by geobalas — APPROVED** [review]

**Pull request merged**